OpenInFirefoxClient/AppDelegate.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  AppDelegate.swift
+//  OpenInFirefoxClient
+//
+//  Created by Bryan Munar on 6/12/15.
+//  Copyright (c) 2015 Bryan Munar. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
```

Comment:
i think we need window to load the buttons and project? i took it out and the app didn't load!